### DISCUSSION TITLE
Storing files with SQL and PRİSMA

### DISCUSSION THREAD

**Discussion by emrekrt1655** [prisma/prisma#13411] [Q&A] [answered]
Is there any way to store jpg files with prisma and SQL in local node js application?

**Comment by nurul3101** [comment, marked as answer]
Hey @emrekrt1655 👋 
Ideally, you should be storing Images in a Blob Store like AWS S3 or similar and just store the reference to it in the SQL Database.
Another alternative is to store images by converting them into base64 format.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
